src/components/Text/text/text.ts

```diff
@@ -71,6 +71,13 @@ export interface TextBaseProps {
      * - text-overflow: ellipsis;
      */
     ellipsis?: boolean;
+    /**
+     * The number of whole lines of text after which the content will be cut off.
+     *
+     * !Note: supports only modern browsers
+     * https://caniuse.com/?search=display%3A%20-webkit-box%3B
+     */
+    ellipsisLines?: 2 | 3 | 4 | 5;
```

Comment:
What if I want more than 5? Maybe let this to be set via JS, in style attribute?

Reply:
added limit by 10 in text utility and unlimited lines count in Text compoennt